plugins/bulk_upload/xml/batch/BulkUploadEngineXml.php

```diff
@@ -2196,6 +2213,21 @@ protected function createEntryFromItem(SimpleXMLElement $item, $type = null)
 		return $entry;
 	}
 	
+	protected function parseAndAddToExistingValues ($fieldName, SimpleXMLElement $item, KalturaBaseEntry $existingItem)
+	{
+		$values = $this->implodeChildElements($item->$fieldName);
+		if($existingItem && strpos($values, self::EXISTING_VALUE_DELIMITER) !== false)
```

Comment:
No need to check existingItem , since function signature validates that it has valid value (unless you meant giving it default=null)